test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -423,7 +423,7 @@ function Test-InstallPackageAPIUnreachableSource
     # Act&Assert
     Assert-Throws {
         [API.Test.InternalAPITestHook]::InstallPackageApi("http://packagesource", "owin", "1.0.0", $false)
-    } "Exception calling `"InstallPackageApi`" with `"4`" argument(s): `"Unable to load the service index for source http://packagesource.`""
+    } "Exception calling `"InstallPackageApi`" with `"4`" argument(s): `"Communication failed to the package source with `"Unable to load the service index for source 'http://packagesource'.`""
```

Comment:
As far as I can tell this updated error message comes from the new resource string `Http_CommunicationFailedWithDetails`, which is unused.  This leads me to believe that this assertion will now fail.

Reply:
Likely so, will address before merge.

Reply:
fixed in latest iteration.